### PR TITLE
Update all react-admin enterprise edition links in the documentation

### PR DESCRIPTION
**Pull request by djhi**
The ra-enterprise website has a new domain. This PR updates all the links in the documentation

### CODE REVIEW

File: docs/AuthRBAC.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: "RBAC"
 
 # Role-Based Access Control (RBAC)
 
-React-admin Enterprise Edition contains [the ra-rbac module](https://marmelab.com/ra-enterprise/modules/ra-rbac)<img class="icon" src="./img/premium.svg" />, which adds fine-grained permissions to your admin. This module extends the `authProvider` and adds replacement for many react-admin components that use these permissions.
+React-admin Enterprise Edition contains [the ra-rbac module](https://react-admin-ee.marmelab.com/modules/ra-rbac)<img class="icon" src="./img/premium.svg" />, which adds fine-grained permissions to your admin. This module extends the `authProvider` and adds replacement for many react-admin components that use these permissions.
```
Comment (adguernier) [inline docs/AuthRBAC.md:8]:
```suggestion
React-admin Enterprise Edition contains [the ra-rbac module](https://react-admin-ee.marmelab.com/documentation/ra-rbac)<img class="icon" src="./img/premium.svg" />, which adds fine-grained permissions to your admin. This module extends the `authProvider` and adds replacement for many react-admin components that use these permissions.
```

File: docs/Buttons.md [hunk 2]
```diff
@@ -818,17 +818,17 @@ const App = () => (
 
 See [The Menu documentation](./Menu.md) for more details.
 
-**Tip**: If you need a multi-level menu, or a Mega Menu opening panels with custom content, check out [the `ra-navigation`<img class="icon" src="./img/premium.svg" /> module](https://marmelab.com/ra-enterprise/modules/ra-navigation) (part of the [Enterprise Edition](https://marmelab.com/ra-enterprise))
+**Tip**: If you need a multi-level menu, or a Mega Menu opening panels with custom content, check out [the `ra-navigation`<img class="icon" src="./img/premium.svg" /> module](https://react-admin-ee.marmelab.com/modules/ra-navigation) (part of the [Enterprise Edition](https://react-admin-ee.marmelab.com))
```
Comment (adguernier) [inline docs/Buttons.md:821]:
```suggestion
**Tip**: If you need a multi-level menu, or a Mega Menu opening panels with custom content, check out [the `ra-navigation`<img class="icon" src="./img/premium.svg" /> module](https://react-admin-ee.marmelab.com/documentation/ra-navigation) (part of the [Enterprise Edition](https://react-admin-ee.marmelab.com))
```

File: docs/Buttons.md [hunk 3]
```diff
@@ -818,17 +818,17 @@ const App = () => (
...
 
 <video controls autoplay playsinline muted loop>
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-multilevelmenu-item.webm" type="video/webm" />
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-multilevelmenu-item.mp4" type="video/mp4" />
+  <source src="https://react-admin-ee.marmelab.com/modules/assets/ra-multilevelmenu-item.webm" type="video/webm" />
```
Comment (adguernier) [inline docs/Buttons.md:824]:
```suggestion
  <source src="https://react-admin-ee.marmelab.com/assets/ra-multilevelmenu-item.webm" type="video/webm" />
```

File: docs/Breadcrumb.md [hunk 4]
```diff
@@ -5,15 +5,15 @@ title: "The Breadcrumb Component"
 
 # `<Breadcrumb>`
 
-This [Enterprise Edition](https://marmelab.com/ra-enterprise)<img class="icon" src="./img/premium.svg" /> component renders a breadcrumb path that automatically adapts to the page location. It helps users navigate large web applications.
+This [Enterprise Edition](https://react-admin-ee.marmelab.com)<img class="icon" src="./img/premium.svg" /> component renders a breadcrumb path that automatically adapts to the page location. It helps users navigate large web applications.
 
 <video controls autoplay playsinline muted loop width="100%">
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-navigation/latest/breadcumb-nested-resource.webm" type="video/webm" />
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-navigation/latest/breadcumb-nested-resource.mp4" type="video/mp4" />
+  <source src="https://react-admin-ee.marmelab.com/modules/assets/ra-navigation/latest/breadcumb-nested-resource.webm" type="video/webm" />
```
Comment (adguernier) [inline docs/Breadcrumb.md:11]:
```suggestion
  <source src="https://react-admin-ee.marmelab.com/assets/ra-navigation/latest/breadcumb-nested-resource.webm" type="video/webm" />
```

File: docs/AccordionForm.md [hunk 5]
```diff
@@ -73,12 +73,12 @@ By default, each child accordion element handles its expanded state independentl
 You can also use the `<AccordionSection>` component as a child of `<SimpleForm>` for secondary inputs:
 
 <video controls autoplay playsinline muted loop>
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-accordion-section-overview.webm" type="video/webm" />
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-accordion-section-overview.mp4" type="video/mp4" />
+  <source src="https://react-admin-ee.marmelab.com/modules/assets/ra-accordion-section-overview.webm" type="video/webm" />
+  <source src="https://react-admin-ee.marmelab.com/modules/assets/ra-accordion-section-overview.mp4" type="video/mp4" />
```
Comment (adguernier) [inline docs/AccordionForm.md:77]:
```suggestion
  <source src="https://react-admin-ee.marmelab.com/assets/ra-accordion-section-overview.mp4" type="video/mp4" />
```

File: docs/AccordionForm.md [hunk 6]
```diff
@@ -5,11 +5,11 @@ title: "AccordionForm"
 
 # `<AccordionForm>`
 
-This [Enterprise Edition](https://marmelab.com/ra-enterprise)<img class="icon" src="./img/premium.svg" /> component offers an alternative layout for Edit and Create forms, where Inputs are grouped into expandable panels.
+This [Enterprise Edition](https://react-admin-ee.marmelab.com)<img class="icon" src="./img/premium.svg" /> component offers an alternative layout for Edit and Create forms, where Inputs are grouped into expandable panels.
 
 <video controls autoplay playsinline muted loop>
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-accordion-form-overview.webm" type="video/webm" />
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-accordion-form-overview.mp4" type="video/mp4" />
+  <source src="https://react-admin-ee.marmelab.com/modules/assets/ra-accordion-form-overview.webm" type="video/webm" />
```
Comment (adguernier) [inline docs/AccordionForm.md:11]:
```suggestion
  <source src="https://react-admin-ee.marmelab.com/assets/ra-accordion-form-overview.webm" type="video/webm" />
```

File: docs/Calendar.md [hunk 7]
```diff
@@ -5,11 +5,11 @@ title: "The Calendar Component"
 
 # `<Calendar>`
 
-This [Enterprise Edition](https://marmelab.com/ra-enterprise)<img class="icon" src="./img/premium.svg" /> component, part of [the `ra-calendar` module](https://marmelab.com/ra-enterprise/modules/ra-calendar), renders a list of events as a calendar. 
+This [Enterprise Edition](https://react-admin-ee.marmelab.com)<img class="icon" src="./img/premium.svg" /> component, part of [the `ra-calendar` module](https://react-admin-ee.marmelab.com/modules/ra-calendar), renders a list of events as a calendar. 
 
 <video controls autoplay playsinline muted loop>
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-calendar.webm" type="video/webm" />
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-calendar.mp4" type="video/mp4" />
+  <source src="https://react-admin-ee.marmelab.com/modules/assets/ra-calendar.webm" type="video/webm" />
```
Comment (adguernier) [inline docs/Calendar.md:11]:
```suggestion
  <source src="https://react-admin-ee.marmelab.com/assets/ra-calendar.webm" type="video/webm" />
```

File: docs/Calendar.md [hunk 8]
```diff
@@ -5,11 +5,11 @@ title: "The Calendar Component"
...
+  <source src="https://react-admin-ee.marmelab.com/modules/assets/ra-calendar.mp4" type="video/mp4" />
```
Comment (adguernier) [inline docs/Calendar.md:12]:
```suggestion
  <source src="https://react-admin-ee.marmelab.com/assets/ra-calendar.mp4" type="video/mp4" />
```

File: docs/Calendar.md [hunk 9]
```diff
@@ -296,11 +296,11 @@ const EventList = () => (
 ```
 {% endraw %}
 
-Check the possible values for `EditDialogProps` in [the `<EditDialog>` component documentation](https://marmelab.com/ra-enterprise/modules/ra-form-layout#createdialog--editdialog).
+Check the possible values for `EditDialogProps` in [the `<EditDialog>` component documentation](https://react-admin-ee.marmelab.com/modules/ra-form-layout#createdialog--editdialog).
 
 ### `CreateDialogProps`
 
-For content addition, `<CompleteCalendar>` relies on [ra-form-layout's `<CreateDialog>`](https://marmelab.com/ra-enterprise/modules/ra-form-layout#createdialog--editdialog) to display its child form in a dialog.
+For content addition, `<CompleteCalendar>` relies on [ra-form-layout's `<CreateDialog>`](https://react-admin-ee.marmelab.com/modules/ra-form-layout#createdialog--editdialog) to display its child form in a dialog.
```
Comment (adguernier) [inline docs/Calendar.md:303]:
```suggestion
For content addition, `<CompleteCalendar>` relies on [ra-form-layout's `<CreateDialog>`](hhttps://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog) to display its child form in a dialog.
```

File: docs/Calendar.md [hunk 10]
```diff
@@ -296,11 +296,11 @@ const EventList = () => (
 ```
 {% endraw %}
 
-Check the possible values for `EditDialogProps` in [the `<EditDialog>` component documentation](https://marmelab.com/ra-enterprise/modules/ra-form-layout#createdialog--editdialog).
+Check the possible values for `EditDialogProps` in [the `<EditDialog>` component documentation](https://react-admin-ee.marmelab.com/modules/ra-form-layout#createdialog--editdialog).
```
Comment (adguernier) [inline docs/Calendar.md:299]:
```suggestion
Check the possible values for `EditDialogProps` in [the `<EditDialog>` component documentation](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog).
```

File: docs/Calendar.md [hunk 11]
```diff
@@ -263,7 +263,7 @@ Check the possible values for `CalendarProps` in the [`<Calendar>`](#calendar) a
 
 ### `EditDialogProps`
 
-For content edition, `<CompleteCalendar>` relies on [ra-form-layout's `<EditDialog>`](https://marmelab.com/ra-enterprise/modules/ra-form-layout#createdialog--editdialog) to display its child form in a dialog.
+For content edition, `<CompleteCalendar>` relies on [ra-form-layout's `<EditDialog>`](https://react-admin-ee.marmelab.com/modules/ra-form-layout#createdialog--editdialog) to display its child form in a dialog.
```
Comment (adguernier) [inline docs/Calendar.md:266]:
```suggestion
For content edition, `<CompleteCalendar>` relies on [ra-form-layout's `<EditDialog>`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog) to display its child form in a dialog.
```

File: docs/ContainerLayout.md [hunk 12]
```diff
@@ -5,11 +5,11 @@ title: "ContainerLayout"
 
 # `<ContainerLayout>`
 
-This [Enterprise Edition](https://marmelab.com/ra-enterprise)<img class="icon" src="./img/premium.svg" /> component offers an alternative to react-admin's `<Layout>` for applications with a limited number of resources. It displays the content in a centered container, has no sidebar, and uses the top bar for navigation.
+This [Enterprise Edition](https://react-admin-ee.marmelab.com)<img class="icon" src="./img/premium.svg" /> component offers an alternative to react-admin's `<Layout>` for applications with a limited number of resources. It displays the content in a centered container, has no sidebar, and uses the top bar for navigation.
 
-![Container layout](https://marmelab.com/ra-enterprise/modules/assets/ra-navigation/latest/container-layout.png)
+![Container layout](https://react-admin-ee.marmelab.com/modules/assets/ra-navigation/latest/container-layout.png)
```
Comment (adguernier) [inline docs/ContainerLayout.md:10]:
```suggestion
![Container layout](https://react-admin-ee.marmelab.com/assets/ra-navigation/latest/container-layout.png)
```

**Review by adguernier — CHANGES_REQUESTED** [review]

File: docs/AccordionForm.md (hunk 6 above)
Comment (djhi) [reply docs/AccordionForm.md:11]: This one works

File: docs/AccordionForm.md [hunk 13]
```diff
@@ -73,12 +73,12 @@ By default, each child accordion element handles its expanded state independentl
 You can also use the `<AccordionSection>` component as a child of `<SimpleForm>` for secondary inputs:
 
 <video controls autoplay playsinline muted loop>
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-accordion-section-overview.webm" type="video/webm" />
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-accordion-section-overview.mp4" type="video/mp4" />
+  <source src="https://react-admin-ee.marmelab.com/modules/assets/ra-accordion-section-overview.webm" type="video/webm" />
```
[the remark this answers is not in the document]
Comment (djhi) [reply docs/AccordionForm.md:76]: This one works

File: docs/AppBar.md [hunk 14]
```diff
@@ -418,8 +418,8 @@ export const MyAppBar = () => (
 A common use case for app bar customization is to add a site-wide search engine. The `<Search>` component is a good starting point for this.
 
 <video controls autoplay playsinline muted loop>
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-search-demo.webm" type="video/webm" />
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-search-demo.mp4" type="video/mp4" />
+  <source src="https://react-admin-ee.marmelab.com/modules/assets/ra-search-demo.webm" type="video/webm" />
```
[the remark this answers is not in the document]
Comment (djhi) [reply docs/AppBar.md:421]: This one works

File: docs/Breadcrumb.md [hunk 15]
```diff
@@ -303,7 +303,7 @@ const MyBreadcrumb = () => (
 
 The `<Breadcrumb.Item>` component is responsible for rendering individual breadcrumb items. It displays the item when the app's location matches the specified `name`. You can nest this component to create breadcrumb paths of varying depths.
 
-![A breadcrumb item](https://marmelab.com/ra-enterprise/modules/assets/breadcrumbItem.png)
+![A breadcrumb item](https://react-admin-ee.marmelab.com/modules/assets/breadcrumbItem.png)
```
[the remark this answers is not in the document]
Comment (djhi) [reply docs/Breadcrumb.md:306]: This one works

File: docs/Buttons.md (hunk 3 above)
Comment (djhi) [reply docs/Buttons.md:824]: This one works

File: docs/AccordionForm.md [hunk 16]
```diff
@@ -73,12 +73,12 @@ By default, each child accordion element handles its expanded state independentl
...
 <video controls autoplay playsinline muted loop>
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-accordion-section-overview.webm" type="video/webm" />
-  <source src="https://marmelab.com/ra-enterprise/modules/assets/ra-accordion-section-overview.mp4" type="video/mp4" />
+  <source src="https://react-admin-ee.marmelab.com/assets/ra-accordion-section-overview.webm" type="video/webm" />
+  <source src="https://react-admin-ee.marmelab.com/assets/ra-accordion-section-overview.mp4" type="video/mp4" />
   Your browser does not support the video tag.
 </video>
 
-Check [the `ra-form-layout` documentation](https://marmelab.com/ra-enterprise/modules/ra-form-layout##accordionform) for more details.
+Check [the `ra-form-layout` documentation](https://react-admin-ee.marmelab.com/documentation/ra-form-layout##accordionform) for more details.
```
Comment (adguernier) [inline docs/AccordionForm.md:81]:
```suggestion
Check [the `ra-form-layout` documentation](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#accordionform) for more details.
```

File: docs/Calendar.md [hunk 17]
```diff
@@ -263,7 +263,7 @@ Check the possible values for `CalendarProps` in the [`<Calendar>`](#calendar) a
 
 ### `EditDialogProps`
 
-For content edition, `<CompleteCalendar>` relies on [ra-form-layout's `<EditDialog>`](https://marmelab.com/ra-enterprise/modules/ra-form-layout#createdialog--editdialog) to display its child form in a dialog.
+For content edition, `<CompleteCalendar>` relies on [ra-form-layout's `<EditDialog>`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog--editdialog) to display its child form in a dialog.
```
Comment (adguernier) [inline docs/Calendar.md:266]: **issue**: does not redirect to the createdialog-editdialog section
```suggestion
For content edition, `<CompleteCalendar>` relies on [ra-form-layout's `<EditDialog>`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog) to display its child form in a dialog.
```

File: docs/Calendar.md [hunk 18]
```diff
@@ -296,11 +296,11 @@ const EventList = () => (
 ```
 {% endraw %}
 
-Check the possible values for `EditDialogProps` in [the `<EditDialog>` component documentation](https://marmelab.com/ra-enterprise/modules/ra-form-layout#createdialog--editdialog).
+Check the possible values for `EditDialogProps` in [the `<EditDialog>` component documentation](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog--editdialog).
```
Comment (adguernier) [inline docs/Calendar.md:299]: **issue**: same
```suggestion
Check the possible values for `EditDialogProps` in [the `<EditDialog>` component documentation](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog).
```

File: docs/Calendar.md [hunk 19]
```diff
@@ -296,11 +296,11 @@ const EventList = () => (
...
 
 ### `CreateDialogProps`
 
-For content addition, `<CompleteCalendar>` relies on [ra-form-layout's `<CreateDialog>`](https://marmelab.com/ra-enterprise/modules/ra-form-layout#createdialog--editdialog) to display its child form in a dialog.
+For content addition, `<CompleteCalendar>` relies on [ra-form-layout's `<CreateDialog>`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog--editdialog) to display its child form in a dialog.
```
Comment (adguernier) [inline docs/Calendar.md:303]: same
```suggestion
For content addition, `<CompleteCalendar>` relies on [ra-form-layout's `<CreateDialog>`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog) to display its child form in a dialog.
```

File: docs/Calendar.md [hunk 20]
```diff
@@ -328,7 +328,7 @@ const EventList = () => (
 ```
 {% endraw %}
 
-Check the possible values for `CreateDialogProps` in [the `<CreateDialog>` component documentation](https://marmelab.com/ra-enterprise/modules/ra-form-layout#createdialog--editdialog).
+Check the possible values for `CreateDialogProps` in [the `<CreateDialog>` component documentation](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog--editdialog).
```
Comment (adguernier) [inline docs/Calendar.md:331]: same 
```suggestion
Check the possible values for `CreateDialogProps` in [the `<CreateDialog>` component documentation](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog).
```

File: docs/CreateInDialogButton.md [hunk 21]
```diff
@@ -27,7 +27,7 @@ npm install --save @react-admin/ra-form-layout
 yarn add @react-admin/ra-form-layout
 ```
 
-**Tip**: [`ra-form-layout`](https://marmelab.com/ra-enterprise/modules/ra-form-layout#createindialogbutton-editindialogbutton-and-EditInDialogButton) is hosted in a private npm registry. You need to subscribe to one of the [Enterprise Edition](https://marmelab.com/ra-enterprise/) plans to access this package.
+**Tip**: [`ra-form-layout`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createindialogbutton-editindialogbutton-and-EditInDialogButton) is hosted in a private npm registry. You need to subscribe to one of the [Enterprise Edition](https://react-admin-ee.marmelab.com/) plans to access this package.
```
Comment (adguernier) [inline docs/CreateInDialogButton.md:30]: Does not scroll the section
```suggestion
**Tip**: [`ra-form-layout`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createindialogbutton-editindialogbutton-and-showindialogbutton) is hosted in a private npm registry. You need to subscribe to one of the [Enterprise Edition](https://react-admin-ee.marmelab.com/) plans to access this package.
```

File: docs/EditInDialogButton.md [hunk 22]
```diff
@@ -27,7 +27,7 @@ npm install --save @react-admin/ra-form-layout
 yarn add @react-admin/ra-form-layout
 ```
 
-**Tip**: [`ra-form-layout`](https://marmelab.com/ra-enterprise/modules/ra-form-layout#createindialogbutton-editindialogbutton-and-EditInDialogButton) is hosted in a private npm registry. You need to subscribe to one of the [Enterprise Edition](https://marmelab.com/ra-enterprise/) plans to access this package.
+**Tip**: [`ra-form-layout`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createindialogbutton-editindialogbutton-and-EditInDialogButton) is hosted in a private npm registry. You need to subscribe to one of the [Enterprise Edition](https://react-admin-ee.marmelab.com/) plans to access this package.
```
Comment (adguernier) [inline docs/EditInDialogButton.md:30]:
```suggestion
**Tip**: [`ra-form-layout`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createindialogbutton-editindialogbutton-and-showindialogbutton) is hosted in a private npm registry. You need to subscribe to one of the [Enterprise Edition](https://react-admin-ee.marmelab.com/) plans to access this package.
```

File: docs/Readme.md [hunk 23]
```diff
@@ -156,7 +156,7 @@ The [React-Admin Enterprise Edition](https://marmelab.com/ra-enterprise) <img cl
 - Get access to exclusive **Learning Material**, including a Storybook full of examples, and a dedicated demo app.
 - Prioritize your needs in the react-admin **Development Roadmap** thanks to a priority vote.
 
-[![React-admin enterprise Edition](https://marmelab.com/ra-enterprise/assets/ra-enterprise-demo.png)](https://marmelab.com/ra-enterprise/)
+[![React-admin enterprise Edition](https://react-admin-ee.marmelab.com/assets/ra-enterprise-demo.png)](https://react-admin-ee.marmelab.com/)
```
Comment (adguernier) [inline docs/Readme.md:159]: *issue**: It seems this asset does not exist in the EE site

File: docs/Reference.md [hunk 24]
```diff
@@ -219,8 +219,8 @@ title: "Index"
 <div class="pages-index" markdown="1">
 
 **- A -**
-* [`useAppLocationState`](https://marmelab.com/ra-enterprise/modules/ra-navigation#useapplocationstate-retrieve-and-define-app-location)<img class="icon" src="./img/premium.svg" />
-* [`useAppLocationMatcher`](https://marmelab.com/ra-enterprise/modules/ra-navigation#useapplocationmatcher-apply-a-matching-on-the-current-app-location)<img class="icon" src="./img/premium.svg" />
+* [`useAppLocationState`](https://react-admin-ee.marmelab.com/documentation/ra-navigation#useapplocationstate-retrieve-and-define-app-location)<img class="icon" src="./img/premium.svg" />
+* [`useAppLocationMatcher`](https://react-admin-ee.marmelab.com/documentation/ra-navigation#useapplocationmatcher-apply-a-matching-on-the-current-app-location)<img class="icon" src="./img/premium.svg" />
```
Comment (adguernier) [inline docs/Reference.md:223]: This anchor does not seem to exist
```suggestion
* [`useAppLocationMatcher`](https://react-admin-ee.marmelab.com/documentation/ra-navigation#useapplocationmatcher)<img class="icon" src="./img/premium.svg" />
```

File: docs/Reference.md [hunk 25]
```diff
@@ -293,11 +293,11 @@ title: "Index"
 * [`useRegisterMutationMiddleware`](./useRegisterMutationMiddleware.md)
 * [`useRemoveFromStore`](./useRemoveFromStore.md)
 * [`useResetStore`](./useResetStore.md)
-* [`useResourceAppLocation`](https://marmelab.com/ra-enterprise/modules/ra-navigation#useresourceapplocation-access-current-resource-app-location)<img class="icon" src="./img/premium.svg" />
+* [`useResourceAppLocation`](https://react-admin-ee.marmelab.com/documentation/ra-navigation#useresourceapplocation-access-current-resource-app-location)<img class="icon" src="./img/premium.svg" />
```
Comment (adguernier) [inline docs/Reference.md:296]: I can't find this section :thinking:

File: docs/Reference.md [hunk 26]
```diff
@@ -293,11 +293,11 @@ title: "Index"
...
 
 **- S -**
 * [`useSaveContext`](./useSaveContext.md)
-* [`useSearch`](https://marmelab.com/ra-enterprise/modules/ra-search#the-usesearch-hook)<img class="icon" src="./img/premium.svg" />
+* [`useSearch`](https://react-admin-ee.marmelab.com/documentation/ra-search#the-usesearch-hook)<img class="icon" src="./img/premium.svg" />
```
Comment (adguernier) [inline docs/Reference.md:300]:
```suggestion
* [`useSearch`](https://react-admin-ee.marmelab.com/documentation/ra-search#usesearch)<img class="icon" src="./img/premium.svg" />
```

**Review by adguernier — CHANGES_REQUESTED** [review]

File: docs/Readme.md (hunk 23 above)
Comment (djhi) [reply docs/Readme.md:159]: I'll fix it on the EE side

File: docs/Reference.md (hunk 25 above)
Comment (djhi) [inline docs/Reference.md:296]:
```suggestion
```

It was removed because it's an internal hook

**Review by adguernier — APPROVED** [review]

**Pull request merged**